P0009/P0009.bs

```diff
@@ -1566,7 +1570,7 @@ struct layout_stride {
     constexpr mapping() noexcept;
     constexpr mapping(mapping const& other) noexcept;
     constexpr mapping(mapping&& other) noexcept;
-    constexpr mapping(Extents e, array&lt;typename Extents::index_type, Extents::rank()> s) noexcept;
+    constexpr mapping(const Extents & e, const array&lt;typename Extents::index_type, Extents::rank()> & s) noexcept;
```

Comment:
there needs to be no space between `Extents` and `&`. Also between `>` and `&`. Also, two lines above this, `const` needs to be on the left of `mapping`. (Just general C++ standard style, like it or not).